scripts/irods/test/resource_suite.py

```diff
@@ -1167,3 +1167,105 @@ def test_itrim_no_op(self):
 
         # try to trim down to repl_count
         self.admin.assert_icommand("itrim -N {repl_count} {filename}".format(**locals()), 'STDOUT_SINGLELINE', "Total size trimmed = 0.000 MB. Number of files trimmed = 0.")
+
+
+    ###################
+    # detached mode
+    ###################
+    def test_detached_mode(self):
+
+        try:
+            file1 = "f1"
+            file2 = "f2"
+            resource_host = "irods.org"
+            resource_name = 'detached_resource'
+
+            hostuser = getpass.getuser()
+
+            resource_context = 'HOST_MODE=detached'
+            testvault = "/tmp/" + hostuser + "/" + resource_name
+
+            self.admin.assert_icommand("iadmin mkresc %s unixfilesystem %s:/%s %s" %
```

Comment:
we have a lib call for this

Reply:
I see lib.create_ufs_resource().  However, it does not set a context.  Would you prefer I use that to create a resource and then modify the resource with assert_icommand()?

I get that you don't want it in /tmp.

Reply:
> However, it does not set a context  

ohhh, i see... that seems like a missing bit of functionality...

we should probably open a new issue for adding context string to that function.

should just be able to add a new parameter, right?   `context=None`?

https://github.com/irods/irods/blob/aea8436ad097e561222cba5f101fa5788d01e1fa/scripts/irods/lib.py#L593